crates/interledger-stream/src/packet.rs

```diff
@@ -1080,6 +1214,46 @@ mod serialization {
         )
     });
 
+    static UNKNOWN_FRAME_PACKET: Lazy<StreamPacket> = Lazy::new(|| {
+        StreamPacketBuilder {
+            sequence: 1,
+            ilp_packet_type: IlpPacketType::try_from(12).unwrap(),
+            prepare_amount: 99,
+            frames: &[Frame::Unknown(UnknownFrameData {
+                frame_type: 89,
+                content: &[1, 2, 3],
+            })],
+        }
+        .build()
+    });
+
+    static SERIALIZED_UNKNOWN_FRAME_PACKET: Lazy<BytesMut> = Lazy::new(|| {
+        BytesMut::from(
+            &vec![
```

Comment:
```suggestion
            &[
```
Vec is probably not needed here...